lib/worker/index.js

```diff
@@ -31,7 +31,7 @@ Object.keys(queues).forEach(name => {
     app.log.info(`Job started name=${name} id=${job.id}`)
   })
 
-  queue.on('complete', function (job, jobPromise) {
+  queue.on('completed', function (job, jobPromise) {
```

Comment:
this is a typo that prevents this log line from coming through.

Reply:
😬